packages/flutter/lib/src/material/data_table.dart

```diff
@@ -205,6 +215,16 @@ class DataRow {
   ///    <https://material.io/design/interaction/states.html#anatomy>.
   final MaterialStateProperty<Color?>? color;
 
+  /// The cursor for a mouse pointer when it enters or is hovering over the
+  /// data row.
+  ///
+  /// If this is null, then the value of [DataTableThemeData.dataRowCursor]
```

Comment:
Should document what states this MaterialStateProperty is resolved against.

Reply:
Update docs.